mask_point_cloud.py

```diff
@@ -0,0 +1,129 @@
+"""Import ZDF point cloud, apply a binary mask, and visualize it."""
+
+import math
+import numpy as np
+import matplotlib.pyplot as plt
+import pptk
+import zivid
+
+
+def _display_rgb(rgb):
+    """Display RGB image.
```

Comment:
Do you think this is needed for the sample code? We want to make the samples as simple and easy to read as possible. I don't think it has to be production quality and super safe. Also, if we introduce checking here then it would be natural to do it elsewhere for other functions.